src/constants/providers.ts

```diff
@@ -18,8 +21,9 @@ class AppJsonRpcProvider extends StaticJsonRpcProvider {
     return this._blockCache
   }
 
-  constructor(urls: string[]) {
-    super(urls[0])
+  constructor(chainId: SupportedChainId) {
+    super(RPC_URLS[chainId][0], { chainId, name: CHAIN_IDS_TO_NAMES[chainId] })
```

Comment:
why did we add the new options object?

Reply:
adding an options object for a StaticJsonRpcProvider allows ethers to skip the initial detectNetwork call to determine _which_ static network the provider fetches for. Considering that this runs during startup, it eases congestion for initial outgoing network requests.